src/main/resources/log4j2.xml

```diff
@@ -0,0 +1,20 @@
+<Configuration status="DEBUG">
+    <Appenders>
+        <Console name="LogToConsole" target="SYSTEM_OUT">
+            <PatternLayout pattern="%d{HH:mm:ss.SSS} [%t] %-5level %logger{36} - %msg%n"/>
+        </Console>
+        <File name="LogToFile" fileName="logs/app.log">
+            <Patern>%d %p %c %m%n</Patern>
```

Comment:
You also forgot to display the number of the line on which this or that information message `<Pattern>%d %p %c:%L %m%n</Pattern>`